docs/release-notes.md

```diff
@@ -6,10 +6,172 @@ search:
 # EasyBuild release notes {: #release_notes }
 
 The latest version of EasyBuild provides support for building and
-installing [**3,470** different software packages](version-specific/supported-software.md),
+installing [**3,552** different software packages](version-specific/supported-software.md),
 including 40 different (compiler) toolchains.
-It contains 255 software-specific easyblocks and 43 generic easyblocks,
-alongside 19,487 easyconfig files.
+It contains 259 software-specific easyblocks and 43 generic easyblocks,
+alongside 19,985 easyconfig files.
+
+
+## EasyBuild v4.9.2 (12 June 2024) {: #release_notes_eb492 }
+
+bugix/update release

```

Comment:
```suggestion
bugfix/update release

```

Reply:
The same typo is also in the release notes of the 4.9.1, 4.8.2, and 4.8.1.

Reply:
fixed with force push in 6ee4943d